File: Assignments/Assignments16/program3.java
class Logic
{
    public void FindFactorial(int iNo)
    {
        int iCnt=0;
        int iFact=1;


        for(iCnt=1;iCnt<=iNo;iCnt++)

        {

            iFact=iFact*iCnt;
         }
           
          System.out.println("Factors are:"+iFact);
    }

}
class Program3

{
    public static void main (String arge[])
    {
        Logic Lobj=new Logic();
        Lobj.FindFactorial(5);

    }
}

File: Assignments/Assignments9/program3.c
#include <stdio.h>

void DisplayEvenFactors(int iNo)
{
    int iCnt = 0;

    if(iNo <= 0)
    {
        iNo = -iNo;   
    }

    printf("Even factors of %d are: ", iNo);

    for(iCnt = 2; iCnt <= iNo; iCnt += 2)   
    {
        if(iNo % iCnt == 0)          
        {
            printf("%d ", iCnt);
        }
    }

    printf("\n");
}

int main()
{
    int iValue = 0;

    printf("Enter number: ");
    scanf("%d", &iValue);

    DisplayEvenFactors(iValue);

    return 0;
}
